apps/api_web/lib/api_web/plugs/validate_date.ex

```diff
@@ -44,15 +44,17 @@ defmodule ApiWeb.Plugs.ValidateDate do
   defp error do
     %{
       code: :no_service,
-      description: "The current rating does not describe service on that date.",
+      detail: "The current rating does not describe service on that date.",
```

Comment:
Was this flagged by Sobelow? Looks like this is the proper format for ErrorSerializer, just curious how you found it.